tests/conftest.py

```diff
@@ -51,6 +53,8 @@
 CONNECTION_CONFIGURATION_SCHEMA = os.path.join(CONFIGURATION_SCHEMA_DIR, "connection-config_schema.json")
 PROTOCOL_CONFIGURATION_SCHEMA = os.path.join(CONFIGURATION_SCHEMA_DIR, "protocol-config_schema.json")
 
+DUMMY_ENV = gym.GoalEnv
```

Comment:
I would have preferred if that was added in the gym tests as it is specific to that. Same for the import. But we can leave it for now.